### ISSUE TITLE
Top level menu's sub menu disabled

### ISSUE THREAD

**Issue by gseok** [webida/webida-client#800]
* This bug occurs in only multi-content editor.
* bug detail
```
 Tool bar's 'save' and 'save all' item is enabled when editor's some content is changed
 But, Top level menu's 'save' and 'save all' item is not enabled.
```
* bug step
```
 1. create 'test.svgx' file in IDE
 2. open 'test.svgx' file with 'Multi content example' editor
 3. draw line (make dirty)
 4. check, tool bar's 'save' and 'save all' item enable
 5. check, top level menu's 'save' and 'save all' item enable

 step 5 abnormal behavior('save' and 'save all' item disabled)
```

* analysis
```
editors/vialble-menu-item.js

if viewer is null then deferred.resolve never called.

bug code...
    function getItemsUnderEdit() {
        logger.info('getItemsUnderEdit()');
        var items = {};
        var deferred = new Deferred();
        var part = _getCurrentEditorPart();
        if (part) {
            var viewer = part.getViewer();
            if (viewer) {
                items = viewer.getMenuItemsUnderEdit(items, menuItems, deferred);
            }
        } else {
            deferred.resolve(items);
        }
        return deferred;
    }
```

**Comment by gseok** [comment]
* fixed code
* https://github.com/webida/webida-client/pull/801

**Comment by hwshim** [comment]
Please refer to #254

**Comment by hwshim** [comment]
Solved in #834
